src/share/poudriere/jail.sh

```diff
@@ -198,6 +198,16 @@ update_jail() {
 	msg "Upgrading using ${METHOD}"
 	case ${METHOD} in
 	ftp|http|ftp-archive)
+		# In case we use FreeBSD dists and TORELEASE is present, check if it's a release branch.
+		if [ ! -z ${TORELEASE} ]; then
```

Comment:
Just use `-n` here rather than `! -z` and please quote _${TORELEASE}_: `-n "${TORELEASE}"`